alacritty/src/display/mod.rs

```diff
@@ -778,6 +779,47 @@ impl Display {
         });
     }
 
+    /// Draw the hyperlink target URI preview of which are currently active.
+    fn draw_active_hyperlink_preview(&mut self, config: &Config, size_info: &SizeInfo) {
+        // Currently we simply stack vi-highlighted and cursor-pointed URL if both exist.
```

Comment:
This phrasing implies we want to change that in the future, why didn't you change that immediately?

Fundamentally I don't think it's too big of a deal to draw overlap if the two URIs should overlap, but chances are we might be able to filter for full equivalency and then just save ourselves the hassle? There's probably not a lot of scenarios where they overlap but aren't exactly the same, right? Not sure if that's possible at all.